README.rst

```diff
@@ -165,6 +192,8 @@ Test the configuration of a running Jenkins instance
 
     make e2e
 
+If e2e fails with broken promise errors, you may need to install firefox-geckodriver as described above in system requirements
```

Comment:
^ I'm not sure if this that helpful. `BrokenPromise` errors in bokchoy are caused when something the test is expecting is not present. Yes, this error was thrown when you encountered issues due to underlying library not being present, but it is misleading. The overwhelming majority of times you see these errors it is because of a test/UI issue. I think that the fact that you mentioned installing geckodriver above should be sufficient